README.md

```diff
@@ -49,6 +51,12 @@ You can run the package script `gulp` to compile and run it on the fly:
 yarn run gulp
 ```
 
+To make commits, you may need to install these packages globally:
```

Comment:
I'm curious: did you find this necessary? I really try to avoid global packages, as our handful of repos is - sadly - at various versions of TypeScript.

Reply:
Ah my mistake. Was fixed more cleanly by adding `tslint` to the devDependencies.